src/metrics/include/metrics/MetricsReporter.hpp

```diff
@@ -25,11 +25,11 @@
 namespace genny::metrics {
 
 /**
- * @namespace genny::metrics::v1 this namespace is private and only intended to be used by genny's
- * own internals. No types from the genny::metrics::v1 namespace should ever be typed directly into
- * the implementation of an actor.
+ * @namespace genny::metrics::internals::v1 this namespace is private and only intended to be used
+ * by genny's own internals. No types from the genny::metrics::internals::v1 namespace should ever
+ * be typed directly into the implementation of an actor.
  */
-namespace v1 {
+namespace internals::v1 {
```

Comment:
The `v1` namespace is already there to indicate it's "internal" - I think this can safely live in just `v1`

Reply:
There were a couple things (like the `OperationImpl` class) that were relevant for both the old metrics system and the new one, while being internal. I tried to refactor the namespaces so that `v1` contains things that are only relevant to the old csv-based system and `v2` is things that are part of the new system, while both are `internal`.

Reply:
(so the `v1` and `v2` namespaces are both inside the `internal` namespace, but some things that are used by both systems are just inside the `internal` namespace)

Reply:
okay that's reasonable. Maybe add a comment somewhere that v1 mostly goes away once we kill csv, and perhaps at that point we rename `internal::v2` to just `v1`.